src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs

```diff
@@ -66,5 +68,14 @@ protected ClassifiedToken[] GetLightbulbPreviewClassification(string menuText)
         {
             return VisualStudio.Editor.GetLightbulbPreviewClassification(menuText);
         }
+
+        protected string PrependLines(string input, string toAdd)
```

Comment:
...what does this do in English? Add a doc comment? The "oh, we don't prepend if the line is blank" seems a bit fishy.